tool/state-processor/src/system/mod.rs

```diff
@@ -211,8 +211,12 @@ impl Processor {
 		account_infos.iter_mut().for_each(|(_, v)| v.data.adjust());
 
 		log::info!("merge solo remaining balances");
-		remaining_ring.into_iter().for_each(|(k, v)| {
+		let (mut total_remaining_ring, mut total_remaining_kton) =
+			(u128::default(), u128::default());
+
+		remaining_ring.into_iter().filter(|(_, v)| *v > 0).for_each(|(k, v)| {
```

Comment:
I think the filter is unnecessary.

Reply:
Why? Most of the items' account under this prefix is zero. We don't need to handle them in mineview.

Reply:
It doesn't make any changes to the following logic.